ci-operator/step-registry/ovn/sdn-migration/feature-migration/config-egress-ip/ovn-sdn-migration-feature-migration-config-egress-ip-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+# I'm unsure if hostsubnet/netnamespace will already exist, maybe i only need to patch them.
```

Comment:
maybe double check and remove the 'oc create' if it's already there, or just change this
comment to say that it may or may not be created already so doing it just in case.

Reply:
also, if you keep it, move it below the NETNAMESPACE_NAME env var creation and use that
variable in the oc create

Reply:
I'll go the latter route.